HashGame.java

```diff
@@ -1,20 +1,30 @@
 import java.util.*;
 
-public class Game {
+public class HashGame {
 
 	public static void main(String[] args) {
-		Player p1 = new Player(1);
-		Player p2 = new Player(2);
+		HashPlayer p1 = new HashPlayer(1);
+		HashPlayer p2 = new HashPlayer(2);
+		HashPlayer sys = new HashPlayer(0);
+		
 		boolean play = true;
 		Scanner s;
 		String r;
+		int round = 1;
 		

```

Comment:
I think we should add comments to explain what's happening here, but otherwise mostly lgtm